COLLADASaxFrameworkLoader/src/COLLADASaxFWLMeshLoader.cpp

```diff
@@ -913,11 +920,13 @@ namespace COLLADASaxFWL
                 mColorList.push_back ( color );
             }
         }
+		return false;
     }
 
     //------------------------------
-    void MeshLoader::initializeTexCoordsOffset ()
+    bool MeshLoader::initializeTexCoordsOffset ()
```

Comment:
Strictly speaking, texcoords are as optional as normals, etc.. The only data that is required is positions. A mesh without texcoords that has a bound texture could have texcoords generated (or even texture ignored) by the importer.